maintenance_equipment_hierarchy/models/maintenance_equipment.py

```diff
@@ -53,7 +53,7 @@ def _compute_complete_name(self):
 
     def preview_child_list(self):
         return {
-            "name": "Child equipment of %s" % self.name,
+            "name": _("Child equipment of %s" % self.name),
```

Comment:
`_("Child equipment of %s") % self.name,

Reply:
Done :wink: